spacy/schemas.py

```diff
@@ -209,6 +210,7 @@ class TokenPatternOperator(str, Enum):
 UnderscoreValue = Union[
     TokenPatternString, TokenPatternNumber, str, int, float, list, bool
 ]
+IobStringValue = Union[TokenPatternString, Literal[Token.iob_strings()]]
```

Comment:
Are you sure this `Literal` works and doesn't mean the whole tuple as the literal? (I've barely used `Literal` before, I'm actually not sure myself.)

Reply:
Yes indeed, I tested a bit on the shell and it worked :grimacing: 

Me too, I looked up from the documentation and examples with `List` and `Tuple` are OK